docs/userguide/networking/get-started-overlay.md

```diff
@@ -71,7 +71,7 @@ store uses the hostnames to identify cluster members.
 Though Docker Machine and Docker Swarm are not mandatory to experience Docker
 multi-host networking with a key-value store, this example uses them to
 illustrate how they are integrated. You'll use Machine to create both the
-key-value store server and the host cluster. This example creates a Swarm
+key-value store server and the host cluster. This example creates a swarm
```

Comment:
note: this is correcting usage of "swarm" for the legacy Docker Swarm, but I think the usage rule that "swarm" is not used as a proper noun still applies.